packages/containers/psammead-timestamp-container/src/timestampUtilities.js

```diff
@@ -1,7 +1,7 @@
 import moment from 'moment-timezone';
 import relativeTime from './relativeTimestamp';
 
-const defaultFormat = 'D MMMM YYYY, HH:mm z';
+const defaultFormat = 'LL, LT z';
```

Comment:
I'd rather leave the default as the UK formatting, and pull in a value from config for each service for their preferred format.

Reply:
@jblaut Changing the format to `LL, LT z` would allow us to have the correct order depending on the locale set. See [this](https://repl.it/repls/EvilStrictDiscussions) example